.changeset/friendly-squids-perform.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/task': major
+---
+
+Do not reset value on status change
```

Comment:
add mention of `error`

Reply:
Can I edit that file directly or do I need to re-run the changeset command?

Reply:
You can just edit the file. The only thing that the changeset command does is generate the file with a nice wizard UI.